test/integration/cart_integration_test.rb

```diff
@@ -14,29 +16,81 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
   test 'an unauthorized user can add an item to the cart' do
     item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
     visit "/items/#{item.id}"
+
     click_link_or_button('Add to Cart')
+
     within('#flash_notice') do
       assert page.has_content?('Added to Cart')
     end
+  end
+
+  test 'an unauthorized user with items in their cart can view their cart' do
+    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.